lab_3_ann_retriever/main.py

```diff
@@ -183,6 +235,16 @@ def vector2tokens(self, vector: Vector) -> list[str] | None:
 
         In case of corrupt input arguments, None is returned.
         """
+        if not vector or len(self._token2ind) != len(vector):
+            return None
+        tokens = []
+        for pose, index in enumerate(vector):
```

Comment:
is it really need to have a loop by `vector`?

Reply:
I guess we can do it the opposite way using self vocab